bitnami/jaeger/templates/query/deployment.yaml

```diff
@@ -0,0 +1,192 @@
+          securityContext: {{- omit .Values.query.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.command "context" $) | nindent 12 }}
+          {{- else if .Values.query.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.query.command "context" $) | nindent 12 }}
+          {{- else }}
+          command: ["/opt/bitnami/jaeger/bin/jaeger-query"]
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" $) | nindent 12 }}
+          {{- else if .Values.query.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.query.args "context" $) | nindent 12 }}
+          {{- else }}
+          {{- end }}
+          {{- if .Values.query.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.query.lifecycleHooks "context" $) | nindent 12 }}
+          {{- end }}
+          ports:
+            - name: ui
+              containerPort: {{ .Values.query.service.ports.api }}
+            - name: admin
+              containerPort: {{ .Values.query.service.ports.admin }}
+
+          env:
+            - name: CASSANDRA_SERVERS
+              value: {{ printf "%s-cassandra" (include "common.names.fullname" .) }}
+            - name: CASSANDRA_PORT
+              value: {{ .Values.cassandra.containerPorts.cql | quote }}
+            - name: CASSANDRA_USERNAME
+              value: {{ .Values.cassandra.dbUser.user }}
+            - name: CASSANDRA_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ printf "%s-cassandra" (include "common.names.fullname" .) }}
+                  key: cassandra-password
+            - name: CASSANDRA_KEYSPACE
+                value: "jaeger_v1_dc1"
+            - name: BITNAMI_DEBUG
+              value: {{ ternary "true" "false" .Values.image.debug | quote }}
+            {{- if .Values.query.extraEnvVars }}
+              {{- include "common.tplvalues.render" (dict "value" .Values.query.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          envFrom:
+            {{- if .Values.query.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ .Values.query.extraEnvVarsCM }}
+            {{- end }}
+            {{- if .Values.query.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ .Values.query.extraEnvVarsSecret }}
+            {{- end }}
+          {{- if not .Values.diagnosticMode.enabled }}
+          {{- if .Values.query.customStartupProbe }}
+          startupProbe: {{- include "common.tplvalues.render" (dict "value" .Values.query.customStartupProbe "context" $) | nindent 12 }}
```

Comment:
What about the other probes?

Reply:
Fixed within the improvements of [a896c22](https://github.com/bitnami/charts/pull/13480/commits/a896c223d6f2e265ad494dd42b6553552d0a89b0)

Probes still pending to refine so still not resolving this conversation.

Reply:
Finalized [7085c8a](https://github.com/bitnami/charts/pull/13480/commits/7085c8a39764644a858a8d1dff77db534c0be787)